app.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const logger = require('morgan');
+const dotenv = require('dotenv');
+const passport = require('passport');
+
+dotenv.config({ path: '.env' });
+
+require('./config/passport');
+require('./models');
+
+const adminController = require('./controllers/admin');
+const userController = require('./controllers/user');
+
+const app = express();
+
+app.set('port', process.env.PORT || 8080);
+
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(express.json());
```

Comment:
Why use both `bodyParser.json()` and `express.json()` middleware?

Reply:
Good catch, `app.use(express.json());` is not necessary. I included body-parser later in the development process and missed that I was using both middlewares.